### PR TITLE
Hugo/bankocr

### PR DESCRIPTION
**Pull request by HugoS922**
Creating Kata for Bankocr

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -33,4 +33,10 @@ task runBankocrAdriana (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.bankocr.adriana.Main'
     standardInput=System.in
     classpath = sourceSets.main.runtimeClasspath
+}
+    
+task runBankOcr (dependsOn: 'classes', type: JavaExec) {
```
Comment (J0c4) [inline build.gradle:38]: This is optional, you don't need to add task to run it for now as we are mainly doing the testing with unit tests

**Review by J0c4 — CHANGES_REQUESTED** [review]
Good job Hugo, tests look good. Try to reduce the size of methods bigger than 5 lines and same for classes bigger than 30 lines. Also, check my comments in some of the files that need to be reverted

**Review by J0c4 — APPROVED** [review]

File: src/test/java/jalau/at18/katas/bankocr/hugo/MainTest.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package jalau.at18.katas.bankocr.hugo;
+
+public class MainTest {
```
[the remark this answers is not in the document]
Comment (HugoS922) [reply src/test/java/jalau/at18/katas/bankocr/hugo/MainTest.java:3]: Hello Jose, I removed the changes in the files Build.gradle and README.

[the remark this answers is not in the document]
Comment (J0c4) [reply src/test/java/jalau/at18/katas/bankocr/hugo/MainTest.java:3]: Thanks!

**Pull request merged**